bin/docker_run.sh

```diff
@@ -12,4 +12,9 @@ if [ $? -ne 0 ]; then
     schematic migrations/
 fi
 
+if [ "$BRAINTREE_MERCHANT_ID" ]; then
```

Comment:
bash is pretty cryptic but shouldn't this be ``if [ -z "$BRAINTREE_MERCHANT_ID" ]`` ?